src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -34,7 +34,8 @@ trait Messages[T <: IncomingEvent, TContext] {
 }
 
 object Messages {
-  val config: Config = ConfigFactory.load
+  private val env = sys.env.getOrElse("ENVIRONMENT", "TEST")
```

Comment:
Is it case insensitive, or should the default be `test`?

Reply:
They are case sensitive but the second value doesn't really matter, its just so that the unit test can run with the application.conf in the test folder.